service/src/main/java/tds/config/repositories/impl/ConfigRepositoryImpl.java

```diff
@@ -0,0 +1,123 @@
+package tds.config.repositories.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.repositories.ConfigRepository;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+@Repository
+public class ConfigRepositoryImpl implements ConfigRepository {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigRepositoryImpl.class);
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ConfigRepositoryImpl(DataSource dataSource) {
+        this.jdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+    }
```

Comment:
At this point, the Config service is only dealing with a single data source.  The current assumption is that the Config service will read WAY more than it will write (it may never write, now that I think about it) so multiple data sources aren't required.  Even if the Config service got its own dedicated AWS Aurora read replica, that's still a single connection, thus a single data source.